src/components/EditHospitalForm/index.js

```diff
@@ -96,6 +100,28 @@ const EditHospitalForm = ({ errors, setErrors, hospital }) => {
           />
         </FormGroup>
 
+        <FormGroup>
+          <Label htmlFor="hospital-survey-url" className="nhsuk-label--l">
+            Key contact survey URL
+          </Label>
+          <span className="nhsuk-hint" id="hospital-survey-url-hint">
+            The survey URL will appear on the visit complete page for the key
+            contact.
+          </span>
+          <Input
+            id="hospital-survey-url"
+            type="url"
+            hasError={hasError("hospital-survey-url")}
+            errorMessage={errorMessage("hospital-survey-url")}
+            className="nhsuk-u-font-size-32"
+            style={{ padding: "16px!important", height: "64px" }}
+            onChange={(event) => setHospitalSurveyUrl(event.target.value)}
+            name="hospital-survey-url"
+            autoComplete="off"
```

Comment:
It might be useful to have autocomplete on for this?

Reply:
Eagle eye! Good catch